app/assets/stylesheets/custom.scss.erb

```diff
@@ -3239,7 +3239,7 @@ label {
     color: #999999;
 }
 .radio span {
-    color: #545454;
+    color: $application-text-color;
```

Comment:
I think the `var(--input-text-color)` for DC is set to a blue color rather than it's $application-text-color so it would change the radio button text for DC to blue rather than keeping it grey.  I could make an additional css variable for the radio buttons (and dropdowns which are handled similarly) instead?

Reply:
Yup, definitely the way to make this more generic